integration/kubernetes/CSI_README.md

```diff
@@ -9,21 +9,24 @@ Kubernetes 1.14 or higher, RBAC enabled in API server(https://kubernetes.io/docs
 ## Usage
 
 
-### Build docker image
+### Docker image
 
-Please run `docker build . -t alluxio/alluxio-csi:<version_tag>` to build CSI docker image. Alluxio doesn't provide official CSI docker image currently.
-You need to build the image by yourself.
+The official `alluxio/alluxio` docker image now supports CSI. By default Kubernetes will pull the latest published `alluxio/alluxio` image on Dockerhub.
+
+Alternatively you can build the docker image yourself following the instructions in `${ALLUXIO_HOME}/integration/docker/README.md`,
+section `Building docker image for production` or `Building docker image for development` depending on your needs. Make sure you refer to your
+built image in the deploying yaml files.
 
 ### Deploy
 
-Please use `helm-generate.sh` to generate related templates. All CSI related templates should under `integration/kubernetes/<deploy-mode>/csi` folder.
+Please use `helm-generate.sh` to generate related templates. All CSI related templates will be under `integration/kubernetes/<deploy-mode>/csi` folder.
 
-You need to deploy `alluxio-csi-controller`, `alluxio-csi-nodeplugin`, `alluxio-csi-driver` before mount volume via CSI.
+You need to deploy `alluxio-csi-controller`, `alluxio-csi-nodeplugin`, `alluxio-csi-driver` before mounting volume via CSI.
 
-We provide two types of provisioning methods. For static provisioning, you need to create `PersistentVolume` and `PersistentVolumeClaim` first.
-For dynamic provisioning, you need to create `StorageClass` and  `PersistentVolumeClaim`.
+We provide two types of provisioning methods. For static provisioning, you need to deploy `alluxio-pv.yaml` and `alluxio-pvc-static.yaml` first.
+For dynamic provisioning, you need to deploy `alluxio-storage-class.yaml` and  `alluxio-pvc.yaml` first.
```

Comment:
Can we put those names as 
`alluxio-pv.yaml (PersistentVolume)` so that it's easier for people not knowing those stuff to understand what those files represent